engine/runtime/src/main/java/org/enso/interpreter/runtime/callable/atom/AtomConstructor.java

```diff
@@ -158,7 +158,7 @@ private void generateQualifiedAccessor() {
             null,
             new FunctionSchema(
                 new ArgumentDefinition(0, "self", ArgumentDefinition.ExecutionMode.EXECUTE)));
-    definitionScope.registerMethod(definitionScope.getAssociatedType(), this.name, function);
+    definitionScope.registerMethod(type.getEigentype(), this.name, function);
```

Comment:
What is **eigentype**? _Own type_?

Reply:
Yeah, the type of this type – where statics live. If you can think of a better name, we can switch